Tests/HederaE2ETests/Token/TokenCreate.swift

```diff
@@ -0,0 +1,393 @@
+           .expirationTime(.now + .minutes(5))
+           .sign(account.key)
+           .execute(testEnv.client)
+           .getReceipt(testEnv.client)
+    }
+
+    internal func testMissingNameFail() async throws {
+       let testEnv = try TestEnvironment.nonFree
+
+       let account = try await makeAccount(testEnv)
+
+        await assertThrowsHErrorAsync(
+            try await TokenCreateTransaction()
+                .symbol("F")
+                .treasuryAccountId(account.id)
+                .expirationTime(.now + .minutes(5))
+                .sign(account.key).execute(testEnv.client),
+                "expected error creating token"
+        ) { error in
+            guard case .transactionPreCheckStatus(let status, transactionId: _) = error.kind else {
+                XCTFail("`\(error.kind)` is not `.missingTokenName`")
+                return
+            }
+
+            XCTAssertEqual(status, .missingTokenName)
+        }
+    }
+
+    internal func testMissingSymbolFail() async throws {
+       let testEnv = try TestEnvironment.nonFree
+
+       let account = try await makeAccount(testEnv)
+
+        await assertThrowsHErrorAsync(
+            try await TokenCreateTransaction()
+                .name("ffff")
+                .treasuryAccountId(account.id)
+                .expirationTime(.now + .minutes(5))
+                .sign(account.key).execute(testEnv.client),
+                "expected error creating token"
+        ) { error in
+            guard case .transactionPreCheckStatus(let status, transactionId: _) = error.kind else {
+                XCTFail("`\(error.kind)` is not `transactionPreCheckStatus`")
+                return
+            }
+
+            XCTAssertEqual(status, .missingTokenSymbol)
+        }
+    }
+
+    internal func testMissingTreasuryAccountIdFail() async throws {
+       let testEnv = try TestEnvironment.nonFree
+
+       let account = try await makeAccount(testEnv)
+
+        await assertThrowsHErrorAsync(
+            try await TokenCreateTransaction()
+                .name("ffff")
+                .symbol("F")
+                .expirationTime(.now + .minutes(5))
+                .sign(account.key)
+                .execute(testEnv.client),
+                "expected error creating token"
+        ) { error in
+            guard case .transactionPreCheckStatus(let status, transactionId: _) = error.kind else {
+                XCTFail("`\(error.kind)` is not `.transactionPreCheckStatus`")
+                return
+            }
+            XCTAssertEqual(status, .invalidTreasuryAccountForToken)
+        }
+    }
+
+    internal func testMissingTreasuryAccountIdSigFail() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        await assertThrowsHErrorAsync(
+            try await TokenCreateTransaction()
+                .name("ffff")
+                .symbol("F")
+                .treasuryAccountId(AccountId.fromString("0.0.3"))
```

Comment:
```suggestion
                .treasuryAccountId("0.0.3")
```
works, alternatively
```suggestion
                .treasuryAccountId(3)
```
or
```suggestion
                .treasuryAccountId(AccountId(num: 3))
```